src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -90,30 +80,32 @@ def create_dialog
       )
     end
 
+    def filesystem_items
+      [
+        Item(Id(Storage::FsType_EXT4), "Ext4"),
+        Item(Id(Storage::FsType_XFS), "XFS"),
+        Item(Id(Storage::FsType_BTRFS), "Btrfs"),
+        Item(Id(Storage::FsType_SWAP), "Swap"),
+        Item(Id(Storage::FsType_NTFS), "NTFS"),
+        Item(Id(Storage::FsType_VFAT), "VFAT")
+      ]
+    end
 
     def doit
-      begin
-
-        log.info "doit #{@blk_device.name}"
+      log.info "doit #{@blk_device.name}"
 
-        if !RemoveDescendantsPopup.new(@blk_device).run()
-          return
-        end
+      return if !RemoveDescendantsPopup.new(@blk_device).run
```

Comment:
`unless`? I do not like unless in multiline if but for trailing one it is encouraged even by suse style guide.